.github/actions/integration-test/action.yaml

```diff
@@ -0,0 +1,45 @@
+name: integration-test
+description: 'action that prepares the integration test setup and runs the provided integration test script'
+inputs:
+  integration-test-script:
+    description: 'file name of integration test script'
+    required: true
+    default: 'integration-test.sh'
+runs:
+  using: "composite"
+  steps:
+    - name: Load Docker images
+      run: |
+        docker load -i ${GITHUB_SHA}_image.tar
+        docker load -i ${GITHUB_SHA}_hook.tar
+      shell: bash
+    - name: Install yq and bash
+      run: |
+        sudo snap install yq
+        sudo apt update
+        sudo apt install bash -y
+      shell: bash
+    - name: Create KinD cluster
+      run: |
+        GO111MODULE="on" go get sigs.k8s.io/kind
+        kind create cluster --wait 120s
+      shell: bash
+    - name: Check KinD cluster
+      run: kubectl get nodes
+      shell: bash
+    - name: Add images to KinD
+      run: |
+        kind load docker-image $(yq e '.deployment.image' helm/values.yaml)
+        kind load docker-image $(yq e '.deployment.helmHookImage' helm/values.yaml)
+      shell: bash
+    - name: Set environment variables for alerting listener
+      run: |
+        SERVICE_CONTAINER_ID=$(docker container ls --no-trunc --format "{{json . }}" | jq ' . | select(.Image|match("alerting-endpoint"))' | jq -r .ID)
```

Comment:
think it might make sense to linebreak here for better readability: https://stackoverflow.com/questions/59954185/github-action-split-long-command-into-multiple-lines
specifically if it is `bash` scripting